troposphere/ecs.py

```diff
@@ -89,6 +89,7 @@ class Service(AWSObject):
         'Cluster': (basestring, False),
         'DeploymentConfiguration': (DeploymentConfiguration, False),
         'DesiredCount': (positive_integer, False),
+        'HealthCheckGracePeriodSeconds': (integer, False),
```

Comment:
Should the validator here be `positive_integer`?

https://github.com/cloudtools/troposphere/blob/master/troposphere/validators.py#L27